repose-aggregator/src/docs/asciidoc/filters/herp.adoc

```diff
@@ -1,3 +1,213 @@
-= Highly Efficient Record Processor Filter
+= Highly Efficient Record Processor (HERP) Filter
```

Comment:
Oh yeah. I probably should have waited for documentation before code reviewing this... I'll take a look tomorrow. 😛